Tests/AfluentTests/WorkerTests/URLSessionAdditionsTests.swift

```diff
@@ -3,7 +3,7 @@
     import Foundation
     import OHHTTPStubs
     import OHHTTPStubsSwift
-    import Testing
+    @_spi(Experimental) import Testing
```

Comment:
Is this still necessary when using swift testing that is shipped with the Swift toolchain? Because CI runs on non-darwin systems. 

It's possible that when you tested it you tested using the now deprecated OSS package

Reply:
I misunderstood, this was to make it work in 15.4

Reply:
This was for Xcode 15.4, but we chatted and decided we don't need to support Xcode 15.4 from the dev side. Removing this and the other place where it's used.